drs_downloader/Gen3DRSClient.py

```diff
@@ -58,74 +58,76 @@ def get_access_url(self, object_id, access_id=None):
 class crdcDRSClient(Gen3DRSClient):
```

Comment:
Re.  these xxxDRSClient classes.  OK to leave in for now?   Let's discuss use case.   Thinking we may want to discard for now and add back in after MVP.